config/ConnectorConfig.json

```diff
@@ -1,15 +0,0 @@
-{

```

Comment:
Not a big deal, but typically this additional commit would be moved to before the `RELEASE: x.y.z` commit.